docs/recipes/Deploying-Modules.md

```diff
@@ -0,0 +1,158 @@
+        restore-keys: |
+          ${{ runner.os }}-node-
+    - name: Install Dependencies
+      run: npm install
+      env:
+        NODE_ENV: development
+    - name: Build Module
+      run: npm run build
+      env:
+        NODE_ENV: production
+    - name: 'Deploy'
+      run: now build --prod --confirm -t $NOW_TOKEN
+      env:
+        NOW_TOKEN: ${{ secrets.NOW_TOKEN }}
+```
+
+If you view the action running, during the `Deploy` step, you should see something like
+
+```bash
+Run now build --prod --confirm -t $NOW_TOKEN
+Now CLI 17.1.1
+- Setting up project
+Linked to [YOUR NOW USERNAME]/[YOUR-REPO-NAME] (created .now and added it to .gitignore)
+Inspect: https://zeit.co/[YOUR NOW USERNAME]/[YOUR-REPO-NAME]/
+- Building
+- Completing
+Production: https://[YOUR-REPO-NAME].now.sh
+https://[YOUR-REPO-NAME]-[HASH].now.sh
+```
+
+## Create and Update Your Module Map
+
+You will now need to add the module you just deployed to your module map.
+
+### Creating a Module Map in Now.sh
+
+To create a module map in Now, you can create a module-map.json locally that looks something like this:
+
+```json
+{
+  "modules": {
+
+  }
+}
+```
+
+You can deploy this by running the [now](https://zeit.co/docs/now-cli#commands/now/basic-usage) command within the directory that you created the `module-map.json` in. You can now view it at the production URL that `now` prints out, you will need that URL for the following steps.
+
+### Create an Update Module Script
+
+You can place this script wherever you would like. However, the following step expects it to be in `scripts/updateModuleMap.js`. If you put it in a different location, make sure you update it in the following step.
+
+```javascript
+const fs = require('fs-extra');
+const fetch = require('node-fetch');
+const { name, version } = require('../package.json');
+
+// This is created during the build process within the deploy action
+const bundleIntegrity = require('../bundle.integrity.manifest.json');
+
+const moduleMapUrl = 'YOUR MODULE MAP URL'; // This is the module map URL you got in the previous step
+
+const STATIC_ASSETS_URL = 'URL TO WHERE YOUR MODULE IS DEPLOYED'; // example 'https://my-module.now.sh'
+
+const updateModuleMap = async () => {
+  try {
+    const response = await fetch(moduleMapUrl);
+
+    const moduleMapContent = await response.json();
+    const dir = 'tmp';
+
+    moduleMapContent.modules[name] = {
+      browser: {
+        url: `${STATIC_ASSETS_URL}/${version}/${name}.browser.js`,
+        integrity: bundleIntegrity.browser,
+      },
+      legacyBrowser: {
+        url: `${STATIC_ASSETS_URL}/${version}/${name}.legacy.browser.js`,
+        integrity: bundleIntegrity.legacyBrowser,
+      },
+      node: {
+        url: `${STATIC_ASSETS_URL}/${version}/${name}.node.js`,
+        integrity: bundleIntegrity.node,
+      },
+    };
+
+    await fs.ensureDir(dir);
+    await fs.writeFile(
+      './tmp/module-map.json', JSON.stringify(moduleMapContent, null, 2)
```

Comment:
not a fan of using the ./tmp folder to output the module map, we should find a more descriptive name for the folder (also the odd chance a CI also uses `tmp` and then you will end up with random artefacts alongside your module map)